### PR TITLE
Add TBSwiftExtensions

### PR DESCRIPTION
**Pull request by tbaranes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ At this time, finding Swift packages seems a bit slim.  Thus, contributions to t
 - [SwiftSequence](https://github.com/oisdk/SwiftSequence) - A framework of extensions for Sequences, inspired by Python's itertools, Haskell's standard library, and other things
 - [SwiftyJSON](https://github.com/SwiftyJSON/SwiftyJSON) - JSON handling in Swift.
 - [swift-parser-generator](https://github.com/pixelspark/swift-parser-generator) - Parser generator
+- [TBSwiftExtensions](https://github.com/tbaranes/TBSwiftExtensions) - A framework of extensions to make swift coding more confortable
```
Comment (donald-pinckney) [inline README.md]: Fix the typo in "confortable"

**Comment by tbaranes** [comment]
Typo fixed

**Comment by donald-pinckney** [comment]
Hhhm, a lot of the functionality seems iOS, OS X, or tvOS specific, maybe it should be moved to the Darwin specific section?

**Comment by tbaranes** [comment]
Most of them are cross-platform, but you are right, since some of them are specific to a platform, I change its section

**Comment by donald-pinckney** [comment]
Hhmm, I'm getting a build error:
```
<unknown>:0: error: filename "AppDelegate.swift" used twice: '/path/to/dir/Packages/TBSwiftExtensions-0.1.3/TBSwiftExtensionsDemo/TBSwiftExtensionDemoOSX/Classes/AppDelegate.swift' and '/path/to/dir/Packages/TBSwiftExtensions-0.1.3/TBSwiftExtensionsDemo/TBSwiftExtensionsDemo/Classes/AppDelegate.swift'
<unknown>:0: note: filenames are used to distinguish private declarations with the same name
```
It seems like its trying to compile your demo code for the library that it will eventually link, which is not ideal anyways.  I'm still getting used to the Package Manager, but my guess would be that if you put your main source code in a top-level `Sources` directory, and the demo code not in there, it would then only compile the stuff in `Sources`.

If you think I'm doing something wrong, please tell me. Also, sorry for being picky about this, I just want everything to work as best it can.

**Comment by tbaranes** [comment]
> If you think I'm doing something wrong, please tell me. Also, sorry for being picky about this, I just want everything to work as best it can.

You are absolutely right about this, it's for the best.

I will check this issueand come back to you once that fixed, thanks for your time and thi library! Like you, I'm still discovering SPM, I probably miss something.

**Comment by donald-pinckney** [comment]
@tbaranes Any update on this?

**Comment by tbaranes** [comment]
I didn't take the time to look at this yet. Will do it soon, if you want, you can close this PR, and I will create a new one once I fix it.

**Comment by tbaranes** [comment]
Hey! Sorry for the delay, I just fix the SPM :)

**Comment by donald-pinckney** [comment]
I just tested it, and it definitely is better.  I have one other problem now though.  If I have:
```
import TBSwiftExtensions
```
in my Swift file, then I get the error:
```
main.swift:15:8: error: no such module 'TBSwiftExtensions'
import TBSwiftExtensions
```

If I change it to: `import ios`, then it compiles great.  But I probably don't really want to include a module just named "ios".
SPM is probably creating a module named "ios" because of the subdirectory "ios" under Sources in your project structure. Though at this point you have played with SPM more than I have.

**Comment by tbaranes** [comment]
Oh, I didn't expect that. Thanks for your feedback, I will check that as soon as possible!

**Comment by tbaranes** [comment]
Closing this issue since I don't have the time to work on it. I will reopen later!

**Pull request closed without merging**